package.json

```
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start",
     "type-check": "tsc",
-    "lint": "eslint pages src",
+    "lint": "eslint '{pages,src}/**/*.{ts,tsx}'",
     "lint:css": "stylelint pages src",
```

Might as well fix this too:
```suggestion
     "lint:css": "stylelint '{pages,src}/**/*.{ts,tsx}'",
```